tests/mocks/fetchers/alerts_fetcher_mock.py

```diff
@@ -317,7 +317,7 @@ def query_pending_source_freshness_alerts(self, *args, **kwargs):
                 detected_at=CURRENT_TIMESTAMP_UTC,
                 snapshotted_at="2022-10-11 10:00:00",
                 max_loaded_at="2022-10-11 10:00:00",
-                max_loaded_at_time_ago_in_s="123123",
+                max_loaded_at_time_ago_in_s=123123,
```

Comment:
I see that we add those fields but do we test that we get the wanted result from them?